addons/binding/org.openhab.binding.netatmo/ESH-INF/i18n/netatmo_de.properties

```diff
@@ -138,3 +153,32 @@ channel-type.netatmo.pressureUnit.description = Zeigt die gew
 channel-type.netatmo.pressureUnit.state.option.0 = mbar
 channel-type.netatmo.pressureUnit.state.option.1 = inHg
 channel-type.netatmo.pressureUnit.state.option.2 = mmHg
+
+channel-type.netatmo.rain.label = Niederschlag
+channel-type.netatmo.rain.description = Zeigt den aktuellen Niederschlag (in mm) an.
+
+channel-type.netatmo.rain1.label = Kumulierter Niederschlag (1h)
+channel-type.netatmo.rain1.description = Zeigt den kumulierten Niederschlag der letzten Stunde (in mm) an.
+
+channel-type.netatmo.rain24.label = Kumulierter Niederschlag (24h)
+channel-type.netatmo.rain24.description = Zeigt den kumulierten Niederschlag der letzten 24h (in mm) an.
+
+channel-type.netatmo.WindAngle.label = Windrichtung
+channel-type.netatmo.WindAngle.description = Zeigt die aktuelle Windrichtung (in �) an.
+
+channel-type.netatmo.WindStrength.label = Windgeschwindigkeit
+channel-type.netatmo.WindStrength.description = Zeigt die aktuelle Windgeschwindigkeit (in km/h) an.
+
+channel-type.netatmo.GustAngle.label = B�en Richtung
+channel-type.netatmo.GustAngle.description = Zeigt die aktuelle B�en Richtung (in �) an.
+
+channel-type.netatmo.GustStrength.label = B�en Geschwindigkeit
+channel-type.netatmo.GustStrength.description = Zeigt die aktuelle B�en Geschwindigkeit (in km/h) an.
+
+channel-type.netatmo.healthindex.label = Health Index
+channel-type.netatmo.healthindex.description = Zeigt den Health Index (healthy, fine, fair, poor, unhealthy) an.
```

Comment:
I think "(healthy, fine, fair, poor, unhealthy)" should be translated, too. Right?